README.md

```diff
@@ -1,11 +1,12 @@
 ## Installation Instructions
 
 1. Navigate to [repo](https://github.com/schvetsov/ArtistDashboard)
-2. Clone locally using `git clone https://github.com/schvetsov/ArtistDashboard.git`
-3. Install dependencies using `npm install`
-4. Run tests using `npm test`
-5. Start the server `npm start`
-6. Navigate to app in [browser](http://localhost:3000)
+```
+git clone https://github.com/schvetsov/ArtistDashboard.git &&
+npm install &&
+npm test &&
+npm start
```

Comment:
The code is missing 'cd ArtistDashboard'

Reply:
Nice catch! implemented in most recent commit.

Reply:
@schvetsov